mne/report.py

```diff
@@ -1158,6 +1271,17 @@ def parse_folder(self, data_path, pattern='*.fif', n_jobs=1, mri_decim=2,
             warnings.warn('`subjects_dir` and `subject` not provided.'
                           ' Cannot render MRI and -trans.fif(.gz) files.')
 
+    def view(self):
+        """Open a current verison of the html report in the web browser
```

Comment:
thinking about this I'd remove this new view method.

just do

report.save('foobar.html', True)

otherwise it will fill up your temp dir.